recipes/protobuf/all/conanfile.py

```diff
@@ -79,30 +131,78 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        os.unlink(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-config-version.cmake"))
+        os.unlink(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-targets.cmake"))
+        os.unlink(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-targets-{}.cmake".format(str(self.settings.build_type).lower())))
+        os.rename(os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-config.cmake"),
+                  os.path.join(self.package_folder, "lib", "cmake", "protobuf", "protobuf-generate.cmake"))
+
+        if not self.options.lite:
+            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "libprotobuf-lite.*")
```

Comment:
dll is not remove here

Reply:
by the way, why not remove (or deprecate) lite option which doesn't exist upstream?

Reply:
Good catch. Thanks!

Removing the option and just packaging everything would be a breaking change for people who link against everything in the package as you can't link against libprotobuf and libprotobuf-lite at the same time. This is a terrible practice, but it is still a breaking chance. And removing options is a breaking change on its own

I think once Protobuf 4 is out we should remove the lite option from the recipe for 4.x.x

Reply:
ok, I understand, so you could remove lite option in package_id instead as @madebr suggested (in the next PR if you want)? It avoids to build the package again, and still prevents to link to both libs (ODR violation?)